drivers/scheduler/k8s/specs/rabbitmq/px-rabbitmq-app.yml

```diff
@@ -168,8 +168,9 @@ spec:
   volumeClaimTemplates:
     - metadata:
         name: rabbitmq-data
+        annotations:
+          volume.beta.kubernetes.io/storage-class: px-rabbitmq-sc
       spec:
-        storageClassName: rabbitmq-sc
```

Comment:
can we fix torpedo to also look for `storageClassName`? using storageClassName in the spec is the right way to specify it. Using annotations was the previous method.

Reply:
Done, found an issue, no need to add annotations here.